### DISCUSSION TITLE
Welcome to LightAwareness Discussions!

### DISCUSSION THREAD

**Discussion by cem-akkaya** [cem-akkaya/LightAwareness#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iceguest2** [comment]
Is there any way to detect a specific light source or maybe color?

**Reply by cem-akkaya** [reply]
Lightsource no since its independent from the source however you can detect color, atleast dominant color simply by;

TArray<FColor> ULightAwarenessComponent::GetBufferPixels()

- GetBufferPixels from blue print as array, for each loop and break color into components. Sum all R , G , B channels into variables and then divide variables to arraylength, will simply return the dominant rgb.

**Reply by cem-akkaya** [reply]
if you need the dominant color feature can add it btw.
